x-pack/test/functional/apps/dashboard/async_search/async_search.ts

```diff
@@ -8,10 +8,16 @@ import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
 export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const es = getService('es');
   const testSubjects = getService('testSubjects');
   const PageObjects = getPageObjects(['common', 'header', 'dashboard', 'visChart']);
 
   describe('dashboard with async search', () => {
+    before(async function () {
+      const { body } = await es.info();
+      if (!body.version.number.includes('SNAPSHOT')) this.skip();
```

Comment:
When we skip, mind logging that we are because we're not running with a snapshot that includes the required shard_delay agg. Might come in useful in the future to people looking at the code or the logs.